os/Kconfig

```diff
@@ -437,6 +444,51 @@ config ARCH_STDARG_H
 		ARCH_STDARG_H=y and providing. If ARCH_STDARG_H, is not defined, then
 		the stdarg.h header file will stay out-of-the-way in include/tinyara/.
 
+config ARCH_HAVE_SETJMP
+        bool
+        default n
+
+config ARCH_SETJMP_H
+        bool "setjmp.h"
+        default n
+        depends on ARCH_HAVE_SETJMP
+        ---help---
+                There is a redirecting version of setjmp.h in the source tree. It
+                resides out-of-the-way at include/nuttx/lib/setjmp.h.  This is
+                because you should normally use your toolchain's setjmp.h file. But
+                sometimes, your toolchain's setjmp.h file may have other header
+                file dependencies and so may not be usable in the NuttX build
+                environment. In those cases, you may have to create a architecture-
+                specific setjmp.h header file at nuttx/arch/<architecture>/include/setjmp.h
+
+                If ARCH_SETJMP_H=y is defined, the top-level makefile will copy the
+                re-directing setjmp.h header file from include/nuttx/lib/setjmp.h to
+                include/setjmp.h. So for the architectures that cannot use their
+                toolchain's setjmp.h file, they can use this alternative by defining
+                ARCH_SETJMP_H=y and providing. If ARCH_SETJMP_H, is not defined, then
+                the setjmp.h header file will stay out-of-the-way in include/nuttx/.
+
+choice
+        prompt "debug.h selection"
```

Comment:
Are we using this? I think we always use debug.h.
If not required, let us remove this config.